### PR TITLE
[POC] Trigger os-autoinst-distri-example tests from fresh openQA

### PR DESCRIPTION
**Pull request by b10n1k**
With this PR a logged in user can trigger a job via a form, providing
settings, git repo and a scenarion definition. User doesnt need to have
special role. Keep in mind that NEEDLES_DIR doesnt need for
os-autoinst-distri-example.
Things which stayed out of this PR are:
- The UI design
- Posibility to upload the yaml definition
- Some way to return data to the webUI, like the job id
- Input validation
- Additional error handling (maybe)

https://progress.opensuse.org/issues/162899

### CODE REVIEW

**Comment by b10n1k** [comment]
https://github.com/user-attachments/assets/b65bf5ed-d37d-49df-9257-0e48d98246cb

Due to the allowed size of upload the footage is as small as possible

File: templates/webapi/NewJob/create.html.ep [hunk 1]
```diff
@@ -0,0 +1,122 @@
+% layout 'bootstrap';
```
Comment (Martchus) [inline templates/webapi/NewJob/create.html.ep:1]: We don't use camel-cased path elements here. So please move this file under `test`. (Is it really that hard to follow the established conventions of a projects. Please don't do everything your way. If everyone would do that this project would end up as a total mess of inconsistencies. **And it becomes really annoying having to state this so often.**)

File: templates/webapi/NewJob/create.html.ep [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+% end
+
+% title 'Run a Job';
+
+<div>
+  <h2><%= title %></h2>
+  
+  %= include 'layouts/info'
+  <p><b>Hint:</b> Experimental form to trigger a test E.g. see <%= link_to 'this
+								      link' => url_for('not_found') %>.
```
Comment (Martchus) [inline templates/webapi/NewJob/create.html.ep:14]: This way of wrapping it is weird:

```suggestion
  <p><b>Hint:</b> Experimental form to trigger a test E.g. see <%= link_to 'this link' => url_for('not_found') %>.
```

File: templates/webapi/NewJob/create.html.ep [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  </p>
+  <div class="container-md">
+  <form class="row g-3" onsubmit="runExample(this);" data-post-url="/api/v1/isos">
```
Comment (Martchus) [inline templates/webapi/NewJob/create.html.ep:17]: The wrapping in the subsequent HTML code is completely inconsistent.

Comment (Martchus) [inline templates/webapi/NewJob/create.html.ep:17]: We should not hardcode paths like `/api/v1/isos`. Use the Mojolicious helper `url_for` instead.

**Review by Martchus — CHANGES_REQUESTED** [review]
I suggest you clean up this PR as mentioned in the inline-comments. Even though it is just a POC we probably want something another person could actually continue to work on. This is not even in a state where it is fun to look at for the sake of reviewing.

I added a few remarks about the modal. However, I suggest you remove it completely and show a flash message instead. That is much easier than making the modal actually work and the modal is not used/working in its current form anyway.

File: templates/webapi/NewJob/create.html.ep [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+machines:
+  64bit:
+    backend: "qemu"
+job_templates:
+  simple_boot:
+    product: "example"
+    machine: "64bit"
+      </textarea>
+    </div>
+
```
Comment (okurz) [inline templates/webapi/NewJob/create.html.ep:92]: I would leave this whole block out for now. openQA should read the scenario definitions from the example test distribution itself.

**Review by okurz — CHANGES_REQUESTED** [review]

File: templates/webapi/NewJob/create.html.ep (hunk 4 above)
Comment (perlpunk) [reply templates/webapi/NewJob/create.html.ep:92]: But having this in the PoC makes it easy to test

File: templates/webapi/NewJob/create.html.ep (hunk 1 above)
Comment (b10n1k) [reply templates/webapi/NewJob/create.html.ep:1]: Now I noticed. believe me if I say I didnt try to do anything on my way here. rather trying to find my way. and what happened briefly I create this somewhere else and then I copied and remained as such because I didnt noticed

**Comment by b10n1k** [comment]
this has been implemented https://github.com/os-autoinst/openQA/pull/5933
